lib/layout/raidlist.css

```diff
@@ -71,6 +71,35 @@
     background-image: url(images/raid_locked.png);
 }
 
+#raidlist .raidSlot .overlayStatus .ui-icon {
+    display: inline-block;
+}
+
+#raidlist .raidSlot .overlayStatus:before {
+    width: 28px;
+    height: 28px;
+    display: inline-block;
+    content: '';
+    margin-right: 16px;
```

Comment:
please see the newer commits, changed to a not-pseudo html element as you suggested and fixed the issue to the size depency